### PR TITLE
Adding declarative ceph shares for frontend and services

### PR DESCRIPTION
**Pull request by muhammadimranfarooqi**

### CODE REVIEW

**Comment by vkuznet** [comment]
Muhammad, please update PR with changes for dmwmmon and dqmgui as I explained in email.

Also, please let me know if you actually applied these changes already in pre-production cluster such that we are sure that they are working with resources you allocated. I just want to double check that with you, i.e. either you tried this out already or you did calculate that we'll have enough resources in pre-production cluster when it will be deployed.

**Comment by muhammadimranfarooqi** [comment]
@vkuznet ,

I have applied changes for dmwmmon and dqmgui as you explained. 

Yes, I have applied those changes for resources in the testbed cluster. 

Best
M Imran

File: kubernetes/cmsweb/services/dbs-default.yaml [hunk 1]
```diff
@@ -1,82 +1,82 @@
...
+            - port: 8250
+              targetPort: 8250
+              name: dbs-default
+            - port: 18250
+              targetPort: 18250
+              name: dbs-default-mon
+            - port: 19250
+              targetPort: 19250
+              name: dbs-d-cpy-mon
+        ---
```
Comment (vkuznet) [inline kubernetes/cmsweb/services/dbs-default.yaml]: Muhammad, are you sure that your editor formatting made a right choice. I see that you indent entire file, e.g. at this line the selector shifted to the right by some (8) spaces. Please remove indentation.

File: kubernetes/cmsweb/services/dbs-default.yaml [hunk 2]
```diff
@@ -1,82 +1,82 @@
...
+                  secretName: dbs-default-secrets
+              - name: hostkey
+                hostPath:
+                    path: /etc/grid-security/hostkey.pem
+                    type: File
+              - name: hostcert
+                hostPath:
+                    path: /etc/grid-security/hostcert.pem
+                    type: File
+        #PROD#- name: logs-cephfs
```
Comment (vkuznet) [inline kubernetes/cmsweb/services/dbs-default.yaml]: here #PROD# are indented to the right

**Pull request merged**